release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -734,6 +734,14 @@ When configuring the `docker/config.json` file to allow pods to pull images from
 
 Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised.
 
+[id="ocp-4-9-poison-pill-operator"]
+==== Enhanced remediation with the Poison Pill Operator 
+
+The Poison Pill Operator now provides following new capabilities:
+
+* Independent of cluster installation type, such as Installer Provisioned Infrastructure (IPI)or User Provisioned Infrastructure (UPI). 
```

Comment:
What does being independent of installation type mean? 
Need space after (IPI)

Reply:
s/Independent of/regardless of
and rephrased it to align it with the description in the main docs content.